apps/meteor/app/livechat/server/api/v1/agent.ts

```diff
@@ -58,3 +60,49 @@ API.v1.addRoute(
 		},
 	},
 );
+
+API.v1.addRoute(
+	'livechat/agent.status',
+	{ authRequired: true, validateParams: isPOSTLivechatAgentStatusProps },
```

Comment:
Should this have view-l-room? That should mean the user is an agent 👀 (since only agents can change status)